streams/src/test/java/org/apache/kafka/streams/integration/SmokeTestDriverIntegrationTest.java

```diff
@@ -96,9 +96,8 @@ SmokeTestDriver.VerificationResult result() {
 
     private static Stream<Boolean> parameters() {
         return Stream.of(
-            // TODO KAFKA-14533: debug and re-enable both parameters
-            Boolean.TRUE
-            //Boolean.FALSE
+            Boolean.FALSE,
```

Comment:
For some additional data I also swapped the order of the two parameters, in case that happens to reveal something about the underlying source of failure here 🤷‍♀️

Reply:
Good idea!